src/actions/eval-frame-action-validator.js

```diff
@@ -19,8 +19,23 @@ const historyActionSchema = Object.assign({}, historySchema);
 historyActionSchema.properties.type = { type: "string" };
 
 const schemas = {
-  ADD_LANGUAGE_TO_EVAL_FRAME: languageActionSchema,
-  APPEND_TO_EVAL_HISTORY: historyActionSchema,
+  ADD_LANGUAGE_TO_EVAL_FRAME: {
```

Comment:
this fixes a console error message that appears every time there is a new plugin loaded.